org.eclipse.jface.text/src/org/eclipse/jface/text/source/SourceViewer.java

```diff
@@ -556,10 +559,22 @@ public void configure(SourceViewerConfiguration configuration) {
 				setDefaultPrefixes(prefixes, t);
 		}
 		setCodeMiningProviders(configuration.getCodeMiningProviders(this));
-
+		installTextViewer();
 		activatePlugins();
 	}
 
+	private void doSetAutoEditStrategies(IAutoEditStrategy[] strategies, String contentType) {
+		super.setAutoEditStrategies(strategies, contentType);
+		if (strategies != null) {
+			for (int i= 0; i < strategies.length; i++) {
```

Comment:
Preferably a `for (IAutoEditStrategy strategy: strategies)` instead of a counted for loop

Reply:
fixed